lms/lmsweb/views.py

```diff
@@ -208,6 +208,51 @@ def share():
     return fail(400, f'Unknown or unset act value "{act}".')
 
 
+@webapp.route('/notes', methods=['GET', 'POST'])
+@login_required
+def note():
+    if not current_user.role.is_manager:
+        return fail(403, "You aren't allowed to access this page.")
+
+    act = request.args.get('act') or request.json.get('act')
+    user_id = request.args.get('userId')
```

Comment:
This will develop to a huge security breach - you shouldn't trust the user sending it's own user id, as it can be faked easily.

Please remove this line and be sure to _never_ trust the frontend when dealing with anything that relates to privileges.

Reply:
Changed it a bit